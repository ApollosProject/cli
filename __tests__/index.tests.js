const { exec } = require('child_process');

test('gets the version', (done) => {
  exec('node index.js -V', (err, out) => {
    expect(out.trim()).toMatchSnapshot();
    done();
  });
});
