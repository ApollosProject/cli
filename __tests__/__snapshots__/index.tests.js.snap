// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gets the version 1`] = `"0.1.2"`;
